test/env.sh

```diff
@@ -9,7 +9,7 @@ export AGENT_LIST="user@host user2@host2"
 export VANILLA_CONTROLLER="user@host"
 
 # Token to access develop versions of Controller
-export PACKAGE_CLOUD_TOKEN=""
+export PACKAGE_CLOUD_TOKEN="3b4ee4b0aac01b954034e1e1c628fcbe7113b299c9934424"
```

Comment:
Is this the public token?

Reply:
Yes, but it is read-only. So it doesn't matter if published